packages/editor/src/components/rich-text/style.scss

```diff
@@ -89,9 +89,18 @@
 		}
 	}
 
-	& + .editor-rich-text__tinymce {
-		opacity: 0.5;
```

Comment:
@iseulde I'm realizing I may be causing regressions by introducing this rule. Can you give me a sanity check that removing this rule? 

What other cases might there be where this opacity would be used as the only way to indicate placeholder text?

Reply:
I'm not sure. Only thing I can think of is adding emoji as placeholders, which would not look like placeholder text.